demo/vl-core.html

```diff
@@ -0,0 +1,24 @@
+
+<!DOCTYPE html>
+<html lang="nl">
+
+<head>
+  <title>vl-search voorbeeld</title>
+  <script type="module" src="/node_modules/vl-ui-demo/dist/vl-demo-all.js"></script>
+
+  <script type="module" src="/src/vl-core.js"></script>
+  <script type="module" src="./vl-slow-element.js"></script>
+
+</head>
+
+<body>
+  <vl-demo-page data-vl-webcomponent="vl-core" data-vl-link="https://overheid.vlaanderen.be/webuniversum/v3/documentation/components/vl-ui-core">
```

Comment:
En ook de link lijkt niet juist, zou eerder https://overheid.vlaanderen.be/webuniversum/v3/documentation/core moeten zijn dan denk ik.

Reply:
klopt